### DISCUSSION TITLE
`types-certifi` and `types-chardet` are no longer needed

### DISCUSSION THREAD

**Discussion by mtelka** [encode/httpx#3014] [Potential Issue] [answered]
From https://pypi.org/project/types-certifi/:
```
Note: The certifi package includes type annotations or type stubs since version 2022.5.18.1.
Please uninstall the types-certifi package if you use this or a newer version.
```
From https://pypi.org/project/types-chardet/:
```
Note: The chardet package includes type annotations or type stubs since version 5.1.0.
Please uninstall the types-chardet package if you use this or a newer version.
```
I believe both `types-certifi` and `types-chardet` should be removed from `requirements.txt`.

**Comment by lovelydinosaur** [comment, marked as answer]
Okay great, thanks. Would you like to issue a pull request and we'll see if we bump into any issues?

**Comment by mtelka** [comment]
Here it is: https://github.com/encode/httpx/pull/3015
